src/scripts/background.js

```diff
@@ -187,11 +187,6 @@ chrome.runtime.onMessage.addListener((msg, sender, sendResponse) => {
         submitCrowns(msg.crowns).then(sendResponse);
         return true;
     }
-
-    if (msg.mhct_golem_submit === 1) {
-        submitGolems(msg.golems).then(sendResponse);
-        return true;
-    }
     // TODO: Handle other extension messages.
```

Comment:
Note that moving this into the main script is why other extensions (like Privacy Badger) can block the request.